SparkPost/support-docs
# WE-422: security vulnerabilities

### What Changed

Fixes security vulns by running npm audit fix.

### How To Test or Verify

- run `npm install`
- verify there are no vulnerabilities
- run `npm run dev` and verify the site still works correctly
- also, verify using the deploy preview link (linked in PR) that the site still works correctly

### PR Checklist
